templateflow/conf/__init__.py

```diff
@@ -76,6 +75,11 @@ def update_home(force=False):
     return False
 
 
-TF_LAYOUT = Layout(
-    str(TF_HOME), validate=False, config='templateflow',
-    ignore=['.git', '.datalad', '.gitannex', '.gitattributes', 'scripts'])
+TF_LAYOUT = None
+try:
+    from .bids import Layout
+    TF_LAYOUT = Layout(
+        str(TF_HOME), validate=False, config='templateflow',
+        ignore=['.git', '.datalad', '.gitannex', '.gitattributes', 'scripts'])
```

Comment:
These may be automatically ignored these days... I might flag them for review in a future PR, though not worth the trouble for this PR, IMO.